modules/retail/pages/retail-install-setup.adoc

```diff
@@ -94,22 +95,28 @@ For instructions to install Salt-based proxy minions and register them to {susem
 
 The activation key should have the following channels:
 
-// FIXME: check this listing. 2019-06-19, ke
 ----
-- SLES 12 SP3 (SP4 in the future) as a base
-  - SLES Pool
-    - SLES Update
-- SUSE Manager 3.2 Tools
-  - SUSE Manager 3.2 Tools Pool
-    - SUSE Manager 3.2 Tools Update
-- SUSE Manager 3.2 Proxy
-  - SUSE Manager 3.2 Proxy Pool
-    - SUSE Manager 3.2 Proxy Update
-- SUSE Manager 3.2 Proxy for Retail
-  - SUSE Manager 3.2 Proxy for Retail Pool
-    - SUSE Manager 3.2 Proxy for Retail Update
+- SUSE Manager Retail Branch Server 4.0 x86_64
+  - Basesystem Module 15 SP1 x86_64
+  - Server Applications Module 15 SP1 x86_64
+    - SUSE Manager Proxy Module 4.0 x86_64
+      - SUSE Manager Retail Branch Server Module 4.0 x86_64
 ----
+// - SUSE Manager 3.2 Tools ???
 
+Pools and respective updates.
+
+// ondrej said:
+////
+sle-product-sles15-sp1-updates-x86_64
+sle-product-suse-manager-retail-branch-server-4.0-updates-x86_64
+sle-module-server-applications15-sp1-updates-x86_64-smrbs-4.0
+sle-module-basesystem15-sp1-updates-x86_64-smrbs-4.0`
+
+sle-manager-tools15-updates-x86_64-sp1
+sle-module-desktop-applications15-sp1-updates-x86_64
+and respective pools
+////
```

Comment:
I was too fast to copy from testsuite.

* sle-product-suse-manager-retail-branch-server-4.0-updates-x86_64
* sle-module-server-applications15-sp1-updates-x86_64-smrbs-4.0
* sle-module-basesystem15-sp1-updates-x86_64-smrbs-4.0

and respective pools are needed for Branch Server proxy.

The rest is for generic SLE15 systems, not sure how we got desktop applications in the mix.